corehq/apps/users/forms.py

```diff
@@ -428,24 +417,10 @@ def __init__(self, *args, **kwargs):
 
     def save(self, user):
         location_id = self.cleaned_data['location']
-        if location_id:
-            loc = Location.get(location_id)
-
-            # This means it will clear the location associations
-            # set in a domain with multiple locations configured.
-            # It is acceptable for now because multi location
-            # config is a not really supported special flag for
-            # IPM.
-            user.set_location(loc)
-
-            # add the supply point case id to user data fields
-            # so that the phone can auto select
-            supply_point = SupplyPointCase.get_by_location(loc)
-            if supply_point:
-                user.user_data['commtrack-supply-point'] = supply_point._id
-
-            user.user_data['commcare_primary_case_sharing_id'] = \
-                LOCATION_SHARING_PREFIX + location_id
```

Comment:
This stuff is a duplication of what happens in `user.set_location`, plus `user` wasn't being saved after calling this anyways.